### PR TITLE
Find an Item by ID and Delete an item

### PR DESCRIPTION
**Pull request by Siyubu**
### What does this PR do?
         find and delete item by id
        

### Description of tasks to be completed
               • add routes for both getting and deleting an item
                • add controllers for both getting and deleting item
### How to test manually
               • Clone the repository or pull from remote and change to "ft-find-item" branch
                 • run npm install && npm run db:init
                  • run npm start and open in your browser http://localhost:4000/api/item/itemId

### CODE REVIEW

**Review by franko-bit — APPROVED** [review]
Good job Solange

**Review by Patience250 — APPROVED** [review]
Good Solange

File: src/controller/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const models = require('../database/models/item')
```
Comment (RedJanvier) [inline src/controller/index.js]:
```suggestion
const models = require('../database/models/index').Item
```

File: src/routes/index.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
 const { Router } = require('express');
 
 const router = Router();
+const controllers = require('../controller')
 
 router.use('/', (req, res) => {
```
Comment (RedJanvier) [inline src/routes/index.js]: I think this route you can remove it too. It was just for test

**Review by RedJanvier — APPROVED** [review]
Amazing work @Siyubu keep it up and take guards to the comments I left...
And please... after fixing the conficts send the PR link to @niyodusengaclement

**Pull request merged**